internal/lcow/disk.go

```diff
@@ -0,0 +1,55 @@
+package lcow
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/Microsoft/hcsshim/internal/log"
+	"github.com/Microsoft/hcsshim/internal/uvm"
+	"github.com/sirupsen/logrus"
+)
+
+// FormatDisk creates a utility vm, mounts the disk as a scsi disk onto to the VM
+// and then formats it with ext4. Disk is expected to be made offline before this
+// command is run. The following powershell commands:
+// 'Get-Disk -Number <disk num> | Set-Disk -IsOffline $true'
+// can be used to offline the disk.
+func FormatDisk(ctx context.Context, lcowUVM *uvm.UtilityVM, destPath string) error {
+	if lcowUVM == nil {
+		return fmt.Errorf("no uvm")
```

Comment:
Nit: Prefer `errors.New()` as there isn't any formatting used here.

Reply:
Keeping it as to keep it similar to the rest of the code in the repo - e.g. disk.go and others.